src/sentry/apidocs/registry.py

```diff
@@ -0,0 +1,932 @@
+from sentry.api.endpoints.accept_organization_invite import AcceptOrganizationInvite
```

Comment:
I also considered having the registry simply be a string to the module/class, but I think having the imports be checked is probably more ergonomic, although it makes the file a bit messier. It can have issues with endpoint name collisions (only 1 currently, two of the JIRA endpoints), so very open to opinions on this.

Reply:
👍 Class objects > strings.

I'd like to try to reduce the redundancy in the imports, though. I think, rather than using `from (module) import (class)`, it would be better to do `import (module)` and list fully qualified class names (e.g., `sentry.api.endpoints.accept_organization_invite.AcceptOrganizationInvite`) down below.

Reply:
As new endpoints are added will they also need to be added to this module?

Reply:
Threw together a quick script to demonstrate what I meant. See https://github.com/getsentry/sentry/pull/31712

Reply:
@markstory if the developer does not document them, yes. But we are pushing for all new endpoints to be documented, so there should be a good reason as to why they'd be added to this module